samples/wfi32iot/src/clicks/pht.c

```diff
@@ -170,16 +137,14 @@ void PHT_readData(float data[], float cal[])
     
     APP_SENSORS_justRead(PHT_I2C_SLAVE_ADDR_RH, 2);
         
-    byte_holder = APP_SENSORS_data.i2c.rxBuffBytes[ 0 ];
-    rh = byte_holder;
-    byte_holder = APP_SENSORS_data.i2c.rxBuffBytes[ 1 ];
-    rh = rh << 8;
-    rh = rh | byte_holder;
-    float rh_flt = (float)rh;
-    rh_flt *= 12500.0;
-    rh_flt /= 65536.0;
-    rh_flt -= 600.0;
-    rh_flt /= 100.0;
+    uint32_t raw_humidity = APP_SENSORS_data.i2c.rxBuffBytes[ 0 ];

```

Comment:
Code style: There should be no spaces between the square bracket and the index value. Comment applies to other code too.